config/crd/bases/serving.kserve.io_inferenceservices.yaml

```diff
@@ -411,7 +410,7 @@ spec:
                               type: array
                           type: object
                       type: object
-                    aix:
+                    alibi:
```

Comment:
@rafvasq -- the changes are not lining up well on GitHub. Did you verify all these changes make sense, when you do a local diff?

Reply:
Hm, well there are quite a few changes on this CRD (1000+ lines) between [the previous copy](https://github.com/kserve/kserve/blob/39ec2d19de1d856a9527b800fda8fb380bc9441f/config/crd/serving.kserve.io_inferenceservices.yaml) and [v0.11.0-rc0](https://github.com/kserve/kserve/blob/v0.11.0-rc0/config/crd/serving.kserve.io_inferenceservices.yaml).